extensions/amp-iframe/1.0/component.js

```diff
@@ -33,8 +34,42 @@ export function Iframe({
   srcdoc,
   ...rest
 }) {
+  const ref = useRef();
+
+  useEffect(() => {
+    const iframe = ref.current;
```

Comment:
`iframe` won't be defined on first render (until after mount)
```suggestion
    const iframe = ref.current;
    if (!iframe) {
      return;
    }
```